### PR TITLE
[BLCR-3875][API] Add private_groups index

### PR DESCRIPTION
**Pull request by Xtimesx**
[BLCR-3875](https://brandslisten.atlassian.net/browse/BLCR-3875)

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -4720,6 +4656,58 @@ paths:
...
                 type: string
+  /private_groups:
+    x-swagger-router-controller: mock
+    get:
+      description: |
+        Request all readable private groups
+      produces:
+        - application/json
+      tags:
+        - privateGroup
```
Comment (marvinjason) [inline swagger.yaml:4667]:
```suggestion
        - PrivateGroup
```

**Comment by marvinjason** [comment]
@Xtimesx I think you updated the wrong file? It got overwritten

File: paths/private_groups.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+x-swagger-router-controller: mock
```
Comment (Xtimesx) [inline paths/private_groups.yaml:1]: @marvinjason this is the new endpoint definition

File: main.yaml [hunk 3]
```diff
@@ -117,6 +117,8 @@ paths:
     $ref: './paths/participation_requests/comments.yaml'
   /personal_feeds/count:
     $ref: './paths/personal_feeds/count.yaml'
+  /private_groups:
```
Comment (Xtimesx) [inline main.yaml:120]: this is the path

File: parameters/Page.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+name: page
```
Comment (Xtimesx) [inline parameters/Page.yaml:1]: this is part of the dry thing

File: parameters/PerPage.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+name: per_page
```
Comment (Xtimesx) [inline parameters/PerPage.yaml:1]: this is part of the dry thing too

**Review by Xtimesx — COMMENTED** [review]
which file do you mean is overwritten?

File: paths/private_groups.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+get:
+  description: |
+    Request all readable private groups
+  produces:
+    - application/json
+  tags:
+    - privateGroup
```
Comment (marvinjason) [inline paths/private_groups.yaml:8]: Change should be here

**Review by marvinjason — APPROVED** [review]

**Pull request merged**